utils/stand_monitoring/poller/poller.py

```diff
@@ -1,88 +1,76 @@
 import json
-
 import polling
 import requests
-from requests import Response
 
 
 config_path = './config.json'
 with open(config_path, 'r') as f_config:
     config = json.load(f_config)
 
 
-def probe(urls: dict, payload: dict, response_patterns: dict):
+def probe(stands: dict, request_timeout: float, response_pattern: int):
```

Comment:
1) Bad naming. We have one stand and several services. `services`, for examle, is better;
2) No need in response pattern, see above.